apps/demos/Demos/Form/GroupedFields/React/App.tsx

```diff
@@ -8,6 +8,14 @@ import service from './data.ts';
 
 const employee = service.getEmployee();
 
+const groupCaptionRender = (iconName) => (data) =>
```

Comment:
it can be optimized more if we create an object with precalculated functions, but for demo current approach is ok i guess